### ISSUE TITLE
Diagnostics still include absolute module paths on macOS

### ISSUE THREAD

**Issue by dinofx** [microsoft/TypeScript#37968]
Seems to be a regression of the fix for #25747, or it was never working as intended on some platforms.

According to #25747, error messages should be showing relative paths to referenced modules.  Instead I'm seeing a chain of `../`s to get back to the root path, followed by the full absolute path.  This is worse that if it just showed the absolute path.  The screenshot below shows a simple workspace with no tsconfig, but I'm seeing this in every workspace, even with tsconfig.json present.

<img width="910" alt="image" src="https://user-images.githubusercontent.com/7581987/79276533-33e35a80-7e76-11ea-9103-eb56c25ee984.png">

**Expected behavior:**

`Module '"./bar"' has no exported member 'bar'.`

**Actual behavior:**

`Module '"../../../Users/rhudson/wtf-ts/bar"' has no exported member 'bar'.`

typescript 3.8.3

VSCode info:

Version: 1.44.1
Commit: a9f8623ec050e5f0b44cc8ce8204a1455884749f
Date: 2020-04-11T01:47:00.296Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 18.7.0

**Comment by dinofx** [comment]
It looks like the fix mentioned above never worked on macOS.  I get absolute paths if I use typescript 3.1.x, but the current behavior starting with typescript 3.2.x.

**Comment by andrewbranch** [comment]
@dinofx I’ve seen this before, but it comes up rarely for me. You can reproduce it consistently? How are you launching VS Code? Does the same thing happen when you invoke `tsc`, or does it only happen in the editor?

**Comment by dinofx** [comment]
The screenshot should have enough details to reproduce this.
The output is the same when using tsc.

**Comment by RyanCavanaugh** [comment]
@dinofx There were some specific questions above that impact how VS Code and TS both behave, can you speak to them?

**Comment by dinofx** [comment]
In a terminal:
```
echo 'import {bar} from "./bar";bar();' > foo.ts
echo 'export function barred(){return "bar";}' > bar.ts
tsc *.ts
```

The output is:
```
foo.ts:1:9 - error TS2305: Module '"../../../Users/rhudson/wtf-ts/bar"' has no exported member 'bar'.

1 import {bar} from "./bar";bar();
          ~~~


Found 1 error.
```

**Comment by andrewbranch** [comment]
This is fixed in TypeScript 3.9.
